### PR TITLE
return the some extra details of the batch

### PR DESCRIPTION
**Pull request by Aadarshikumar**

### CODE REVIEW

File: lib/services/partners.js [hunk 1]
```diff
@@ -1350,29 +1350,40 @@ module.exports = class PartnerService extends Schmervice.Service {
...
-      const response = {}
 
-      response.title = classDetails[0].title
-      response.recurring_id = recurring_id
-      response.start_time = classDetails[0].start_time;
-      response.end_time = classDetails[0].end_time
-      response.name = name
-      return [null, [response]];
+      const partner_Specific = await PartnerSpecificBatches.query().throwIfNotFound().where('recurring_id', recurring_id);
+      const pathway_data = await PathwaysV2.query().where('id', partner_Specific[0].pathway_id);
```
Comment (giribabu22) [inline lib/services/partners.js:1361]: What if in the partner_Specific array there is not data. Example empty array or there is no pathway_id

File: lib/services/partners.js [hunk 2]
```diff
@@ -1350,29 +1350,40 @@ module.exports = class PartnerService extends Schmervice.Service {
...
-      response.end_time = classDetails[0].end_time
-      response.name = name
-      return [null, [response]];
+      const partner_Specific = await PartnerSpecificBatches.query().throwIfNotFound().where('recurring_id', recurring_id);
+      const pathway_data = await PathwaysV2.query().where('id', partner_Specific[0].pathway_id); 
+      const facilitator_details = await User.query().throwIfNotFound().where('id', classDetails[0].facilitator_id);
+      
+      let batchStartDate = new Date(classDetails[0].start_time);
+      let lastClasses = classDetails.length-1;
+      let bacthEndDate = new Date(classDetails[lastClasses].end_time);
```
Comment (giribabu22) [inline lib/services/partners.js:1366]: What if the second last class will resudule the date after the last class. It will not run i guess.

Comment (Aadarshikumar) [reply lib/services/partners.js:1366]: can you again review the PR. now it will work.

**Comment by giribabu22** [comment]
Done.

**Pull request merged**